scripting/practicemode/commands.sp

```diff
@@ -586,3 +586,67 @@ public Action Command_Break(int client, int args) {
   PM_MessageToAll("Broke all breakable entities.");
   return Plugin_Handled;
 }
+
+public Action Command_TestDatabaseConnection(int client, int args) {
+  if (!g_InPracticeMode) {
+    return Plugin_Handled;
+  }
+
+  if (!GetCvarIntSafe("sm_practicemode_use_database")) {
+    PM_Message(client, "sm_test_database (.testdb) requires sm_practicemode_use_database to be enabled.");
```

Comment:
You can just use g_UseDatabaseCvar.BoolValue here. GetCvarIntSafe is only really useful when getting the handle to a cvar built into the game where it's not worth saving the handle with FindConVar.